src/adapters/postgres/rbac/privilege-adapter.ts

```diff
@@ -53,19 +57,16 @@ export class PostgresPrivilegeService {
         privileges.push(new PrivilegeResponseDto(response));
       }
     } catch (e) {
-      return new ErrorResponseTypeOrm({
-        statusCode: HttpStatus.INTERNAL_SERVER_ERROR,
-        errorMessage: e.message || "Internal server error",
-      });
+      return APIResponse.error(response, apiId, "Internal Server Error", `Error is ${e}`, HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
@poojakarma 
       const errorMessage = e.message || "Internal server error";
      return APIResponse.error(response, apiId, "Internal Server Error", errorMessage, HttpStatus.INTERNAL_SERVER_ERROR);

Reply:
@poojakarma Follow this for all the error messages thrown in this file

Reply:
ok mam